File: GymWare-Backend/GymWare.Entities/DTO/RutinaEjerciciosDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using GymWare.Entities;

namespace GymWare.Entities.DTO
{
    public class RutinaEjerciciosDTO
    {
        public List<RutinaEjercicio> RutinaEjercicios { get; set; }
        public Rutina Rutina { get; set; }
        
    }
}

File: GymWare-Frontend/src/app/classes/rutinaEjerciciosDTO.ts
import { Rutina } from "src/app/classes/rutina";
import { RutinaEjercicio } from "./rutinaEjercicio";

export class RutinaEjerciciosDTO {

    Rutina: Rutina;
    RutinaEjercicios: RutinaEjercicio[] = [];

    constructor() {}
}